Cargo.toml

```diff
@@ -26,9 +26,4 @@ wasm-bindgen-futures = "0.4"
 web-sys = {version = "0.3.22", default-features = false}
 
 [dev-dependencies]
-bevy = {version = "0.9", default-features = false, features = [
-  "bevy_asset",
-  "png",
-  "render",
-  "x11", # GitHub Actions runners don't have libxkbcommon installed, so can't use Wayland
-]}
+bevy = "0.10"
```

Comment:
Please don't include more features than necessary

Reply:
Oh, i didn't realize it was dev deps at first... Still it will build faster in ci and dev without all the extra stuff